romeo/asset-contract/contracts/asset.clar

```diff
@@ -67,17 +76,20 @@
     (tree-depth uint)
     (block-header (buff 80)))
     (begin
-        (try! (is-contract-owner))
+        (asserts! (is-contract-owner) err-forbidden)
+        (asserts! (> amount u0) err-bad-request)
         (try! (verify-txid-exists-on-burn-chain withdraw-txid burn-chain-height merkle-proof tx-index tree-depth block-header))
-        (print {notification: "burn", payload: withdraw-txid})
         (ft-burn? sbtc amount owner)
+        (print {notification: "burn", payload: withdraw-txid})
+    		(ok true)
     )
 )
 
 ;; #[allow(unchecked_data)]
 (define-public (transfer (amount uint) (sender principal) (recipient principal) (memo (optional (buff 34))))
 	(begin
-		(asserts! (or (is-eq tx-sender sender) (is-eq contract-caller sender)) err-not-token-owner)
+		(asserts! (is-contract-owner) err-forbidden)
```

Comment:
assets can't be transferred? I am confused.

Reply:
is fixed now